source/auth/auth.md

```diff
@@ -375,7 +375,7 @@ When connected to MongoDB 3.2 or earlier:
 
 - source
 
-    MUST be "$external". Defaults to `$external`.
+    MUST be "$external". Defaults to `$external\`.
```

Comment:
Looking at the rich diff, this adds the backslash at the end of the code block. Is that expected?

Reply:
This is incorrectly rendered in GitHub:
<img width="391" height="79" alt="Image" src="https://github.com/user-attachments/assets/3647c0e2-4956-4e09-9399-85b7d1235eaa" />

Reply:
Updated, now only the logging file has changed